Not working with Gradle 7

After upgrading to gradle 7.0 the build fails because the old version `2.0.0` of `com.gorylenko.gradle-git-properties` is used.

A workaround is to the newer version of the plugin directly into the project:
```
plugins {
    id "com.gorylenko.gradle-git-properties" version "2.2.4"
}
```

The fix should be to upgrade `com.gorylenko.gradle-git-properties` to version `2.2.4`.

I'll open a PR for it.

The error looks like:
```
> Task :generateGitProperties FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Some problems were found with the configuration of task ':generateGitProperties' (type 'GenerateGitPropertiesTask').
  - Type 'GenerateGitPropertiesTask' property 'gitProperties' is missing an input or output annotation.
    
    Reason: A property without annotation isn't considered during up-to-date checking.
    
    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#missing_annotation for more details about this problem.
  - Type 'GenerateGitPropertiesTask' property 'source' is annotated with @InputFiles but missing a normalization strategy.
    
    Reason: If you don't declare the normalization, outputs can't be re-used between machines or locations on the same machine, therefore caching efficiency drops significantly.
    
    Possible solution: Declare the normalization strategy by annotating the property with either @PathSensitive, @Classpath or @CompileClasspath.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#missing_normalization_annotation for more details about this problem.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
```

--- a/build.gradle
+++ b/build.gradle
@@ -27,11 +27,11 @@ repositories {
 dependencies {
   implementation gradleApi()
   implementation localGroovy()
-  implementation 'gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:2.0.0'
+  implementation 'gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:2.2.4'
 }
 
 group = 'com.pasam.gradle.buildinfo'
-version = '0.1.3'
+version = '0.1.4'
 
 targetCompatibility = '1.8'
 sourceCompatibility = '1.8'

